src/features/user/ManageProjects/styles/StepForm.module.scss

```diff
@@ -169,18 +169,27 @@
 
 .inlineLinkButton {
   font-family: $primaryFontFamily;
-  width: 100%;
-  font-size: 14px;
+ width: 100%;
+ font-size: 14px;
   font-weight: normal;
   font-stretch: normal;
   font-style: normal;
   line-height: 1.43;
   letter-spacing: normal;
-  text-align: center;
   color: #68b030;
+  display: flex;
+  justify-content: center;
+  margin-top: 10px;
 }
 .inlineLinkButton:hover {
   cursor: pointer;
+  font-weight: bold;
```

Comment:
Why are we changing weight on hover ?

Reply:
Because the hover effect was not visible.